config/jobs/kubernetes-sigs/boskos/boskos-presubmits.yaml

```diff
@@ -7,7 +7,7 @@ presubmits:
       path_alias: sigs.k8s.io/boskos
       spec:
         containers:
-          - image: public.ecr.aws/docker/library/golang:1.22.3
+          - image: public.ecr.aws/docker/library/golang:1.22.5
```

Comment:
TODO: switch to go toolchain?

for a project like boskos we can probably trivially adopt the go.mod statement?

Reply:
ACK i see a lot more stuff to be done in boskos as well :(